src/app/components/create-edit-event-page/create-edit-event-page.component.ts

```diff
@@ -161,30 +159,35 @@ export class CreateEditEventPageComponent implements OnInit {
                     this.tags.push({ tagName: tag });
                 }
             }
+            
+            // By pathcing a form, validators are not 'executed'.
+            // Since endTime, joinDeadlineDate and joinDeadlineTime controls are injected with a custom error, 
+            //they should be marked as touched to trigger the validator.
+            this.eventGeneralForm.get('eventDates').get('endTime').markAsTouched();
+             this.eventGeneralForm.get('eventDates').get('joinDeadlineTime').markAsTouched();
+            this.eventGeneralForm.get('eventDates').get('joinDeadlineDate').markAsTouched();
 
         })
     }
 
     initEventFormControls(): void {
         this.eventGeneralForm = new FormGroup({
-            title: new FormControl('', [Validators.required]),
+            title: new FormControl('', [Validators.required, Validators.maxLength(15)]),
```

Comment:
15 seems like too less characters. 
"Cooking for Charity" -is already 19.
I thing 25 would be better.